instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-arquillian-testing/jaxrs-2.0-arquillian-testing.gradle

```diff
@@ -5,7 +5,9 @@ apply from: "$rootDir/gradle/java.gradle"
 
 // add repo for org.gradle:gradle-tooling-api which org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-gradle-depchain depends on
 repositories {
+  mavenCentral()
```

Comment:
Why do we need to add this to individual projects? I thought that `settings.gradle` will take care of that?

Reply:
Unfortunately it seems `repositories` overrides what's in `settings.gradle`, not add to it. So for the few times we need custom repos, we need to define them all. It's not too common so I guess it's OK